libretro-common/file/config_file.c

```diff
@@ -617,6 +646,16 @@ static int config_file_from_string_internal(
    return 0;
 }
 
+void config_file_set_reference_path(config_file_t *conf, char *path)
+{
+   if (conf)
+   {
+      /* Assumes if you wanted a relative path the input path  is
+       * already relative to the config
```

Comment:
Not a good assumption... :)

Since this is libretro-common code, I think we should do it 'properly' - i.e. use a resolve relative path function here.